.vscode/settings.json

```diff
@@ -27,7 +27,7 @@
   // https://github.com/microsoft/vscode-cpptools/issues/722
   "C_Cpp.autoAddFileAssociations": false,
   "C_Cpp.default.cppStandard": "c++17",
-  "lldb.library": "/Applications/Xcode.app/Contents/SharedFrameworks/LLDB.framework/Versions/A/LLDB",
```

Comment:
@jtbandes I made this change to get debugging to work on my mac without installing Xcode - maybe this would be a good change? it means users can debug with only the command-line tools installed.

Reply:
👍  on the other hand, if someone doesn't have them installed and only has Xcode, this will probably break? But this is fine